src/kubernetes/stream.rs

```diff
@@ -35,6 +35,8 @@ where
             }
         }
         decoder.finish().map_err(|data| Error::UnparsedDataUponCompletion { data })?;
+
+        ()
```

Comment:
@MOZGIII I had some trouble figuring out what this looked like expanded, but I think it is a clippy false-positive resulting from it thinking that we could simply do:

```
         decoder.finish().map_err(|data| Error::UnparsedDataUponCompletion { data })
```

instead, but that doesn't compile in this context. I resolved by simply returning `()` at the end instead, but I'm curious what you think.

Reply:
The lint error was:

```
warning: unneeded `return` statement    
   --> src/kubernetes/stream.rs:21:5 
    |                                                
21  | /     try_stream! {                            
22  | |         let mut decoder: MultiResponseDecoder<T> = MultiResponseDecoder::new();
23  | |                                              
24  | |         debug!(message = "Streaming the HTTP body.");
...   |                                              
37  | |         decoder.finish().map_err(|data| Error::UnparsedDataUponCompletion { data })?;
38  | |     }                                        
    | |     ^ in this macro invocation              
    | |_____|                                        
    |                                                
    |                                                
   ::: /home/jesse/.cargo/registry/src/github.com-1ecc6299db9ec823/async-stream-impl-0.3.0/src/lib.rs:210:1
    |                                                
210 | / pub fn try_stream(input: TokenStream) -> TokenStream {
211 | |     let mut stmts = match parse_input(input) {
212 | |         Ok(x) => x,                                                                               
213 | |         Err(e) => return e.to_compile_error().into(),
...   |                                                                                                   
242 | |     .into()                                                                                       
243 | | }                                            
    | |_- in this expansion of `try_stream!`
    |                                                                                                     
    = note: `#[warn(clippy::needless_return)]` on by default                     
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_return
```

Reply:
@jszwedko I removed `()` which you added and do not see any clippy errors, am I doing something wrong?

Reply:
@fanatid ah, you are right! This only shows up with `+nightly` which I was using to make use of `cargo clppy --fix`. I'll drop it.